toolkits/nature/packages/nature-header/scss/50-components/header.scss

```diff
@@ -53,11 +52,25 @@
 	margin: 0 (0 - spacing(8)); // Make flush with edge
 }
 
-.c-header__menu--primary {
+.c-header__menu--global {
+	justify-content: flex-end;
+}
+
+.c-header__menu--global .c-header__item {
+	svg {
+		display: none;
+
+		@include media-query('sm') {
+			display: block;
+		}
+	}
+}
+
+.c-header__menu--journal {
 	font-weight: $font-weight-bold;
 }
 
-.c-header__menu--secondary {
+.c-header__menu--updates {
```

Comment:
What does 'updates' mean in the header?

Reply:
it was an attempt to group the sign up for alerts and rss feed links together

Reply:
I was trying to group that mean and the journal menu as the same 'thing'. That's why I went with secondary. Stylistically they're the same.

Reply:
But if you're keen to split them, maybe something like `--tools` (as we moved them from journal tools section